_layouts/main.html

```diff
@@ -62,15 +62,22 @@ <h2>Meetings</h2>
 
 
   <div class="col-lg-4">
-    <h2>Hacktoberfest</h2>
+    <h2>Software Citation and Recognition</h2>
     <p>
-      The <a href="https://hacktoberfest.com/">Hacktoberfest</a> is now running again. 
-      Contribute to selected repositories from HSF Training and Scikit-HEP and earn a T-shirt or plant a tree.
-      Click <a href="https://github.com/issues?q=is%3Aissue+is%3Aopen+archived%3Afalse+sort%3Aupdated-desc+label%3Ahacktoberfest+org%3Ahsf-training+org%3Ascikit-hep+org%3Ahsf">here</a> to find all eligible issues!
+      How do we cite software in HEP and how do we give people proper recognition? It's 
+      a vital question for our community.
     </p>
-    <img src="{{ '/images/Hfest-Logo-2-Color-Void@2x.png' | relative_url }}"
-    alt="Hacktoberfest-logo" style="display: block; width:100%; margin-left: auto; margin-right: auto;">
-   </div>
+    <p>
+      The HSF and IRIS-HEP are organising a (virtual) mini-workshop to discuss just that,
+      with discussion on current practices from the experiments,
+      the view from developers, and also input from Zenodo and Inspire.
+    </p>
+    <p>
+      Register now
+      for the <a href="https://indico.cern.ch/event/1211229/">workshop</a> to
+      give your input and help us decide on what we believe are best practices to follow.
```

Comment:
that sounds odd. "Help us to decide what we believe". What about "help us to find out on which practices would be most useful to bring our field and community forward". Only much later after the process we can start to "believe".